### DISCUSSION TITLE
management page loads, but after enetering password ask for authorization

### DISCUSSION THREAD

**Discussion by chadleywilson** [rabbitmq/rabbitmq-server#9743] [Questions] [answered]
Hi All,
I have installed RabbitMQ using the bitnami charts.
this is my install command

```
helm install rabbitmq-is bitnami/rabbitmq \
--set replicaCount=1 \
--set volumePermissions.enabled=true \
--set clustering.forceBoot=true \
--set podManagementPolicy=Parallel \
--set auth.username=admin,auth.password=mypass \
--set auth.enableLoopbackUser=false \
--set namespaceOverride=mynamespace \
--set readinessProbe.enabled=false \
--set persistence.existingClaim=pvc-rabbitmq
```
It installs and works fine
The PODs create the environment looks great. we have the same install on other clusters. no issues

but this cluster something is different.
After deployment I edit the service and change it to LoadBalancer. I am using metallb.

I navigate to the ip http://10.0.1.120:15672 and then login appears.

I enter the username admin and the password from my command and then the best way to explain it an edge windows pops up saying
Sign in to access this site
Authorization required by http://10.0.1.120:15672
Your connection to this site is not secure
then has fields for username and passwords

I have not been able to search what it is or where it comes from?

I have added
`--set management.tcp.ip=0.0.0.0` 
But more than this I don't know what the issue could be.

**Comment by michaelklishin** [comment]
Enabling [HTTP API request logging](https://rabbitmq.com/management.html#http-logging) and tacking a traffic capture are the best ways to narrow the issue down. My guess is that something prevents the UI from storing the identity/credentials it uses for later requests. But I cannot tell you what specifically that may be.

If this was an upgrade, clearing all local state (caches, local storage) in your browser may help.

**Reply by chadleywilson** [reply]
I won't lie, I am new to RabbitMQ never used it, or had to install it before. (it seems to be a quite the complex beast)

But I can't imagine in its simplest form its not  any different to any other system.

I assume if it presents its web management UI then it is "Good to Go"?

What I am trying to ask is could the issue "based on your experience", more likely  be our environment rather than something to do with the RabbitMQ deployment?

For interest sake - here is pic of the login prompt:
![image](https://github.com/rabbitmq/rabbitmq-server/assets/43165291/289aaaaa-9c45-482d-b0dd-df91aeb5e791)

**Comment by chadleywilson** [comment, marked as answer]
> Hi All, I have installed RabbitMQ using the bitnami charts. this is my install command
> 
> ```
> helm install rabbitmq-is bitnami/rabbitmq \
> --set replicaCount=1 \
> --set volumePermissions.enabled=true \
> --set clustering.forceBoot=true \
> --set podManagementPolicy=Parallel \
> --set auth.username=admin,auth.password=mypass \
> --set auth.enableLoopbackUser=false \
> --set namespaceOverride=mynamespace \
> --set readinessProbe.enabled=false \
> --set persistence.existingClaim=pvc-rabbitmq
> ```
> 
> It installs and works fine The PODs create the environment looks great. we have the same install on other clusters. no issues
> 
> but this cluster something is different. After deployment I edit the service and change it to LoadBalancer. I am using metallb.
> 
> I navigate to the ip http://10.0.1.120:15672 and then login appears.
> 
> I enter the username admin and the password from my command and then the best way to explain it an edge windows pops up saying Sign in to access this site Authorization required by http://10.0.1.120:15672 Your connection to this site is not secure then has fields for username and passwords
> 
> I have not been able to search what it is or where it comes from?
> 
> I have added `--set management.tcp.ip=0.0.0.0` But more than this I don't know what the issue could be.

I just thought I would post the solution to this.

After much looking online, I did stumble on a few suggestions. But really the issue for me was that I had used Azure File share for my persistent volume.
There are resources locks in Azure. When I tried to delete all the files in the share. I couldn't.

So I created a new share, recreated the PV (and PVC for good measure). 
Reran the helm command above.

This time it works perfectly.

( I removed the resource locks and deleted the old share, I could have kept it. but it was quicker to  test with a new share)
